lib/routes/digitalcameraworld/news.ts

```diff
@@ -0,0 +1,73 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import parser from '@/utils/rss-parser';
+import { load } from 'cheerio';
+import { ofetch } from 'ofetch';
+const host = 'https://www.digitalcameraworld.com';
+export const route: Route = {
+    path: '/news',
+    categories: ['new-media'],
+    example: '/digitalcameraworld/news',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['digitalcameraworld.com/'],
+        },
+    ],
+    name: 'News',
+    maintainers: ['EthanWng97'],
+    handler,
+};
+
+async function handler() {
+    const rssUrl = `${host}/feeds.xml`;
```

Comment:
The feed already provides full articles with output much cleaner than this PR.

Reply:
seems like newsblur can't parse dc:content

Reply:
is there a way to push this changes? seems like some reader can't parse dc:content

Reply:
You can replace `description` with `dc:content`

Reply:
the item.content provided by rss-parser is not correct

Reply:
Use cheerio with [xmlMode](https://cheerio.js.org/docs/api/interfaces/CheerioOptions).

Reply:
done